Tools/i18n/pygettext.py

```diff
@@ -163,15 +163,12 @@
 import time
 import getopt
 import ast
-import token
 import tokenize
+from collections import defaultdict
+from dataclasses import dataclass, field
+from operator import attrgetter, itemgetter
 
-__version__ = '1.5'
-
-default_keywords = ['_']
-DEFAULTKEYWORDS = ', '.join(default_keywords)
-
-EMPTYSTRING = ''
+__version__ = '1.6'
```

Comment:
I bumped the version since this adds some new capabilities, but let me know if it's not needed!

Reply:
Good point, I reverted that change. My original reasoning was that since the version is written to the POT file we might want to bump it up but I agree that it should use the Python version itself, not a separate version.